crypto/evp/evp_fetch.c

```diff
@@ -146,21 +135,22 @@ static void destruct_method(void *method, void *data)
 }
 
 void *evp_generic_fetch(OPENSSL_CTX *libctx, int operation_id,
-                        const char *algorithm, const char *properties,
```

Comment:
why was this changed to name?

Reply:
For consistency.  We have "name", "algorithm_name" and "algorithm".  Since this is EVP, we *know* they are names of algorithms, and this parameter *is* the algorithm name, so...